pkg/scheduler/objects/preemption.go

```diff
@@ -873,10 +873,7 @@ func preemptPredicateCheck(plugin api.ResourceManagerCallback, ch chan<- *predic
 func batchPreemptionChecks(checks []*si.PreemptionPredicatesArgs, batchSize int) [][]*si.PreemptionPredicatesArgs {
 	var result [][]*si.PreemptionPredicatesArgs
 	for i := 0; i < len(checks); i += batchSize {
-		end := i + batchSize
-		if end > len(checks) {
-			end = len(checks)
-		}
+		end := min(i+batchSize, len(checks))
```

Comment:
`i + batchSize`

Reply:
I wanna ask. Is there some format command we can call?
I use Intellij format command to format and also notice it is a littlte different. 
So I try to search `i+` , I see 
<img width="527" alt="image" src="https://github.com/apache/yunikorn-core/assets/20961779/ef04661c-ef22-4081-8e06-e12f0af7e8c5">
The format don't have space too, so I ..

Reply:
it seems there is no strict format :_